locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -3275,6 +3273,9 @@ msgstr "Vorhaben bearbeiten"
 msgid "Project Links"
 msgstr "Projektlinks"
 
+#: meinberlin/apps/plans/exports.py:52
+msgid "Modified"
+
```

Comment:
Ups, during our rebasing, we lost the empty `msgstr ""`. I hope that doesn't break anything.